pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/NamespaceOperation.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pulsar.common.policies.data;
+
+/**
+ * Namespace authorization operations.
+ */
+public enum NamespaceOperation {
+    CREATE_TOPIC,
```

Comment:
nit: it would be good to group similar operations together

```
    CREATE_TOPIC,
    GET_TOPIC,
    GET_TOPICS,
    DELETE_TOPIC,

    ADD_BUNDLE,
    DELETE_BUNDLE,
    GET_BUNDLE,

    GET_PERMISSION,
    GRANT_PERMISSION,
    REVOKE_PERMISSION,

    CLEAR_BACKLOG,
    UNSUBSCRIBE,
```

Reply:
You mean merge GET_TOPIC and GET_TOPICS, yes. Other points?

Reply:
I mean organizing the operations in the way I just proposed in previous comment.

Reply:
Oh i thought it was quote, nvm, on it.

Reply:
done.